binding/android/OrcaTestApp/orca-test-app/src/androidTest/java/ai/picovoice/orca/testapp/BaseTest.java

```diff
@@ -89,35 +98,55 @@ public static String[] getModelFiles() {
         };
     }
 
-    protected static float getWordErrorRate(
-            String transcript,
-            String expectedTranscript,
-            boolean useCER) {
-        String splitter = (useCER) ? "" : " ";
-        return (float) levenshteinDistance(
-                transcript.split(splitter),
-                expectedTranscript.split(splitter)) / transcript.length();
+    public static short[] concatArrays(short[] existingArray, short[] arrayToAdd) {
```

Comment:
not that this really matters since it's just the test, but this should be `protected` or `private` since we're only using it in the test classes